src/Sylius/Component/Core/spec/Promotion/Filter/ProductFilterSpec.php

```diff
@@ -53,4 +53,9 @@ function it_returns_all_items_if_configuration_is_invalid(OrderItemInterface $it
     {
         $this->filter([$item], [])->shouldReturn([$item]);
     }
+
+    function it_returns_all_items_if_configuration_is_empty(OrderItemInterface $item)
```

Comment:
The problem is, that is it the default behaviour of of promotion configuration. The configuration is present, but it is empty, because filter form type is included in an action form type. We need it with a current solution.